net-misc/rsync-bpc/rsync-bpc-3.1.3.0-r1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Rsync-bpc is a customized version of rsync that is used as part of BackupPC"
+HOMEPAGE="https://github.com/backuppc/rsync-bpc"
+SRC_URI="https://github.com/backuppc/rsync-bpc/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="virtual/ssh"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+		"${FILESDIR}/3.1.3.0/${P}-fix-gettimeofday-error.patch" #874666"
```

Comment:
That quote at the end shouldn't be there. The path/versioning also looks a bit weird here? Why bother with the version in the path if the patch is named with `${P}`? But also, I usually recommend only using `${P}` if you think it's in the next upstream release.

Reply:
What would you recommend regarding the path/versioning? Removing the folder structure completely and just putting the version in the name of the patch? What should I rather do if I think its not in the next upstream version?

Reply:
Thanks I will stick with ${P}. Just looked it up and it has already been merged :)

Reply:
Done